polars/polars-io/src/json.rs

```diff
@@ -242,7 +242,7 @@ where
 
 #[cfg(test)]
 mod test {
-    use std::io::Cursor;
+    use std::{ffi::OsStr, io::Cursor};
```

Comment:
Can we move the tests to the integration tests? I want to have most test there as it is easier on compile times.